src/modules/Dashboard/features/Applet/Schedule/Calendar/Calendar.styles.ts

```diff
@@ -104,6 +104,19 @@ export const StyledCalendarWrapper = styled(Box, shouldForwardProp)`
         padding: ${({ hasMoreBtn }: { hasMoreBtn: boolean }) =>
           hasMoreBtn ? theme.spacing(1.1, 0, 2.1) : theme.spacing(1.1, 0, 0.9)};
       }
+
+      .rbc-event:not(.rbc-event-allday) {
+        max-width: 96% !important;
```

Comment:
Is it worth to keep it in a variable (if we have the same `max-width`/`width` rules  in calendars)?